docs/guides/getting-started/integrate-tauri.mdx

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 3
+pagination_next: guides/architecture/README
+---
+
+import TauriInitQuestions from './_fragments/_tauri-init-questions.mdx'
+import TauriFiles from './_fragments/_tauri-files.mdx'
+import Commands from './_fragments/_commands.mdx'
+import { InstallTauriCLI, InstallTauriAPI } from '@theme/Command'
+import TauriInit from './_fragments/_tauri-init.mdx'
+
+# Integrate into Existing Project
+
+If you already have an existing web project, this guide will walk you through integrating Tauri into your project, whether it is Node.js-based (like [Svelte], [React], [Vue], or [Angular]) or Rust-based (like [Yew] or [Dominator]).
+
+:::info
+
+Before we continue, make sure you have completed the [prerequisites] to have a working development environment.
+
+:::
+
+For the purposes of this guide we will use a [React] project that has been created using the [`create-react-app`][create-react-app] tool and has a similar structure to this:
+
+```
+│── README.md
+│── package.json
+│── public
+│   │── favicon.ico
+│   ╰── index.html
+╰── src
+    │── App.css
+    │── App.js
+    │── index.css
+    ╰── index.js
+```
+
+## Create the Rust Project
+
+<TauriInit
+  destDirValue="../build"
+  devPathValue="https://localhost:3000"
+  beforeDevCommandValue="yarn start"
+  beforeBuildCommandValue="yarn build"
+/>
```

Comment:
@FabianLars saw your comment on #855, what do you mean incorrect? these values are based on the `create-react-app` template, ofc they will be incorrect for other templates but the guide can't cover them all. Do you have a better suggestion on how we can handle this?